src/main/java/Timo/Timo.java

```diff
@@ -0,0 +1,584 @@
+package Timo;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+import java.util.*;
+import java.io.File;
+import java.time.format.DateTimeFormatter;
+
+class Task {
```

Comment:
Consider separating the classes into different files? e.g. Task.java, Todo.java etc